src/components/Textfields/TextFieldBase/TextfieldBase.stories.tsx

```diff
@@ -0,0 +1,66 @@
+import TextfieldBase, { TexfieldBaseProps } from ".";
+
+import { Story, Meta, StoryContext } from "@storybook/react";
+
+const backgroundColor = {
+    light: "#fff",
+    dark: "#000",
+};
+
+const styles = (context: StoryContext) => {
+    return {
+        backgroundColor:
+            context.args.color === "secondary" ? backgroundColor.dark : backgroundColor.light,
+        minHeight: "300px",
```

Comment:
Please use the real theme instead
https://material-ui.com/styles/api/#usetheme-theme